extensions/ui/iconset/org.eclipse.smarthome.ui.iconset.classic/README.md

```diff
@@ -3,1521 +3,83 @@
-    <img src="icons/pressure.png" alt="pressure" title="pressure">
-    <figcaption>pressure</figcaption>
-  </figure>
-  <figure style="width: 128px; display: inline-block; text-align: center; font-size: 0.8em; margin: 16px 8px;">
-    <img src="icons/projector_benq.png" alt="projector_benq" title="projector_benq">
-    <figcaption>projector_benq</figcaption>
-  </figure>
-  <figure style="width: 128px; display: inline-block; text-align: center; font-size: 0.8em; margin: 16px 8px;">
-    <img src="icons/pump.png" alt="pump" title="pump">
-    <figcaption>pump</figcaption>
-  </figure>
-  <figure style="width: 128px; display: inline-block; text-align: center; font-size: 0.8em; margin: 16px 8px;">
-    <img src="icons/qualityofservice.png" alt="qualityofservice" title="qualityofservice">
-    <figcaption>qualityofservice</figcaption>
-  </figure>
-  <figure style="width: 128px; display: inline-block; text-align: center; font-size: 0.8em; margin: 16px 8px;">
-    <img src="icons/qualityofservice-0.png" alt="qualityofservice-0" title="qualityofservice-0">
-    <figcaption>qualityofservice-0</figcaption>
-  </figure>
-  <figure style="width: 128px; display: inline-block; text-align: center; font-size: 0.8em; margin: 16px 8px;">
-    <img src="icons/qualityofservice-1.png" alt="qualityofservice-1" title="qualityofservice-1">
-    <figcaption>qualityofservice-1</figcaption>
-  </figure>
-  <figure style="width: 128px; display: inline-block; text-align: center; font-size: 0.8em; margin: 16px 8px;">
-    <img src="icons/qualityofservice-2.png" alt="qualityofservice-2" title="qualityofservice-2">
-    <figcaption>qualityofservice-2</figcaption>
-  </figure>
-  <figure style="width: 128px; display: inline-block; text-align: center; font-size: 0.8em; margin: 16px 8px;">
-    <img src="icons/qualityofservice-3.png" alt="qualityofservice-3" title="qualityofservice-3">
-    <figcaption>qualityofservice-3</figcaption>
-  </figure>
-  <figure style="width: 128px; display: inline-block; text-align: center; font-size: 0.8em; margin: 16px 8px;">
-    <img src="icons/qualityofservice-4.png" alt="qualityofservice-4" title="qualityofservice-4">
-    <figcaption>qualityofservice-4</figcaption>
-  </figure>
-  <figure style="width: 128px; display: inline-block; text-align: center; font-size: 0.8em; margin: 16px 8px;">
-    <img src="icons/radiator.png" alt="radiator" title="radiator">
-    <figcaption>radiator</figcaption>
-  </figure>
-  <figure style="width: 128px; display: inline-block; text-align: center; font-size: 0.8em; margin: 16px 8px;">
-    <img src="icons/rain.png" alt="rain" title="rain">
-    <figcaption>rain</figcaption>
-  </figure>
-  <figure style="width: 128px; display: inline-block; text-align: center; font-size: 0.8em; margin: 16px 8px;">
-    <img src="icons/receiver.png" alt="receiver" title="receiver">
-    <figcaption>receiver</figcaption>
-  </figure>
-  <figure style="width: 128px; display: inline-block; text-align: center; font-size: 0.8
```

Comment:
All icons from the channel categories are still contained in that list and not filtered out as expected.